operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -167,6 +167,29 @@ groups:
     for: 1h
     labels:
       severity: warning
+  - alert: MimirRulerInstanceHasNoRuleGroups
```

Comment:
I think you mentioned it in a Slack conversation but I underestimated it. This alert could be very noisy in the read-write deployment mode (or monolithic one). I would explicitly target the microservices mode here, which is that we care about because if you run monolithic or RW then this is not an issue anymore because the same process runs multiple components so you're not wasting resources anyway.

> I would explicitly target the microservices mode here

How? Filter on `$._config.instance_names.ruler` (look at other queries doing it).

Reply:
Fixed 👍